handlers/zebedee.go

```diff
@@ -10,10 +12,9 @@ import (
 
 // ZebedeeClient is an interface for zebedee client
 type ZebedeeClient interface {
-	SetAccessToken(string)
-	GetBreadcrumb(string) ([]data.Breadcrumb, error)
-	Get(string) ([]byte, error)
-	GetDatasetLandingPage(string) (data.DatasetLandingPage, error)
-	GetDataset(string) (data.Dataset, error)
+	GetBreadcrumb(context.Context, string) ([]data.Breadcrumb, error)
+	Get(context.Context, string) ([]byte, error)
```

Comment:
`Get` what? The other methods are specific - get breadcrumb, Dataset landing page etc. Feel like this should have a more descriptive name like ` GetData` or `GetContentJSON` or whatever it is it does. It makes it clearer at a glance what it's doing without having look into the method specifics

Reply:
Not something i've added - i've only removed methods. `Get` is very generic in this case. You could use it to get breadcrumbs, dataset landing page etc. as they all wrap `get`. `GetData` is a tad more accurate but still somewhat vague. It's not something i've added, so it can be a debate for another time